app/views/posts/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> New Post </h1>
+
+<%= notice || alert %>
```

Comment:
This  should be placed in the application layout

Reply:
done